staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -699,3 +704,28 @@ func (p *processorListener) setResyncPeriod(resyncPeriod time.Duration) {
 
 	p.resyncPeriod = resyncPeriod
 }
+
+// isObjectSame returns true if oldObj and newObj have same ResourceVersion.
+func isObjectSame(oldObj, newObj interface{}) bool {
+	oldRO, ok := oldObj.(runtime.Object)
+	if !ok {
+		return false
+	}
+
+	newRO, ok := newObj.(runtime.Object)
+	if !ok {
+		return false
+	}
+
+	oldRV, err := meta.NewAccessor().ResourceVersion(oldRO)
```

Comment:
can we construct a single accessor outside the function as a private package var and reuse it?